src/cmd/policy.go

```diff
@@ -54,6 +62,15 @@ opslevel run policy -f policy.rego | jq
 	},
 }
 
+func RegoFuncReadFile(ctx rego.BuiltinContext, a *ast.Term) (*ast.Term, error) {
+	if str, ok := a.Value.(ast.String); ok {
+		contents, err := os.ReadFile(string(str))
```

Comment:
do we need to do filepath exists validation here?

Reply:
Ya i'm not entirely sure here - whats the user experience like when you use a policy with a file thats not present?

Reply:
It exits with the error so it doesn't read the other files.  Changed it to warn if filepath doesn't exist.